ddtrace/contrib/molten/__init__.py

```diff
@@ -13,8 +13,8 @@ def hello(name: str, age: int) -> str:
 
     ddtrace-run python app.py
 
-To enable distributed tracing when using autopatching, set the
-``DD_MOLTEN_DISTRIBUTED_TRACING`` environment variable to ``True``.
+To disable distributed tracing when using autopatching, set the
```

Comment:
In general I think we should not advice how to disable distributed tracing. But to move forward, I would say:
1. change it ON by default (this PR)
2. Have @andrewsouthard1 to look at our docs and give a huge revamp